apps/fabric-website/src/pages/Styles/TypographyPage/docs/web/TypographyCustomization.md

```diff
@@ -0,0 +1,29 @@
+You can add some of your own customizations to the default theme.
+For example, you can utilize `defaultFontStyle` to override the default font style and
+`fonts` to override the default font styles for specific font variants, like small, medium, large, etc.
+The example below shows how to override font styles for some of the font variants.
+You can override anything in `IRawStyle`.
```

Comment:
I wonder if saying something like this (and ideally linking to IRawStyle docs) would be better:
```
The overrides can include any property from `IRawStyle`.
```

Though I also noticed that this doesn't seem to be entirely true--for example, when I tried changing the color, that override didn't seem to be respected.

Reply:
Oh interesting, I'll investigate that

Reply:
So any `IRawStyle` should work as long as that component's styles file did not specify that style. I think 'color' may not have worked for the component you tested it on, because that component's styles specify a color. Is that the case?

For example, I tried the following on a Checkbox with a label:

```tsx
const theme = loadTheme({
  fonts: {
    medium: {
      fontSize: '30px',
      fontWeight: 'bold',
      paddingTop: '30px',
      color: 'red'
    }
  }
});
```

I think the only reason color is not being applied in this case is because it's specified in the styles for text:

https://github.com/OfficeDev/office-ui-fabric-react/blob/79a55b109a2e062de932e45e6e16d1e9c1f485b3/packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts#L231-L234

Not every component uses `theme.fonts` in its styling, but those that do should allow for this customization.